includes/widgets/social-icons.php

```diff
@@ -533,6 +541,34 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_group_control(
+			Group_Control_Box_Shadow::get_type(),
+			[
+				'name' => 'social_icon_hover_shadow',
+				'selector' => '{{WRAPPER}} .elementor-social-icon:hover',
+			]
+		);
+
+		$this->add_control(
+			'social_icon_transition',
```

Comment:
It seems that whenever a transition is used in a hover (which it most likely would always be) it conventionally gets the word `_hover_` in the name too.

Have a look and confirm this if you can.

I this case we should stick to this and follow this in the whole PR too e.g.
`social_icon_transition` becomes `social_icon_hover_transition`

Tanks
`